From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Armbian <info@armbian.com>
Date: Tue, 7 Jan 2025 00:00:00 +0000
Subject: [PATCH] Set bootcmd for Armbian boot script

Configure U-Boot to load and run Armbian boot script from partition
instead of using legacy raw sector boot.

Signed-off-by: Armbian <info@armbian.com>
---
 configs/ma35d1_sdcard1_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/configs/ma35d1_sdcard1_defconfig b/configs/ma35d1_sdcard1_defconfig
index aaaaaaa..bbbbbbb 100644
--- a/configs/ma35d1_sdcard1_defconfig
+++ b/configs/ma35d1_sdcard1_defconfig
@@ -11,7 +11,8 @@ CONFIG_ARMV8_SET_SMPEN=y
 CONFIG_DISTRO_DEFAULTS=y
 # CONFIG_ARCH_FIXUP_FDT_MEMORY is not set
 CONFIG_BOOTDELAY=3
-# CONFIG_USE_BOOTCOMMAND is not set
+CONFIG_USE_BOOTCOMMAND=y
+CONFIG_BOOTCOMMAND="mmc dev 0; ext4load mmc 0:1 0x85010000 /boot/boot.scr && source 0x85010000; run distro_bootcmd"
 # CONFIG_CONSOLE_MUX is not set
 CONFIG_BOARD_LATE_INIT=y
 CONFIG_SYS_PROMPT="MA35D1> "
--
2.34.1
